examples/v1/pipelineruns/beta/http-resolver-credentials.yaml

```diff
@@ -0,0 +1,35 @@
+# This http resolver example will uses a username and password to access the
+# URL.
+#
+# http-password-secret is a Kubernetes secret containing the
+# password in the same namespace where this PipelineRun runs.
+---
+kind: Secret
+apiVersion: v1
+metadata:
+  name: my-secret
+stringData:
+  token: "token"
+---
+apiVersion: tekton.dev/v1
+kind: PipelineRun
+metadata:
+  generateName: http-resolver-
+spec:
+  pipelineSpec:
+    tasks:
+      - name: http-resolver
+        taskRef:
+          resolver: http
+          params:
+            - name: url
+              value: https://api.hub.tekton.dev/v1/resource/tekton/task/tkn/0.4/raw
```

Comment:
sorry a naive question, is this a public file? Do we need credentials to fetch it?

Reply:
it is a public file, but it's okay to pass credentials to this url since it would not error out and this tests that credentials are passed. I you have a way to manage a endpoint URL on the tekton infra to have it running just for so we can run that examples as intended then please let me know but that may be a bit more work for not much benefits.

Reply:
I see, thanks. This is fine.